### PR TITLE
Update experiments-msmarco-passage.md

### PR DESCRIPTION
**Pull request by Stefan824**
Error met:
ImportError: libmkl_intel_lp64.so.1: cannot open shared object file: No such file or directory It appears that the latest version of Faiss had some issues with handling dependencies. Error solved by downgrading faiss to the 2021 version. Done by running command $conda install mkl=2021

System Information:
conda 23.9.0
Operating System: Ubuntu 22.04.3 LTS on WSL
Python Version: Python 3.10.2
Java Version:
openjdk 11.0.20.1 2023-08-24
OpenJDK Runtime Environment (build 11.0.20.1+1-post-Ubuntu-0ubuntu122.04) OpenJDK 64-Bit Server VM (build 11.0.20.1+1-post-Ubuntu-0ubuntu122.04, mixed mode, sharing) Maven Version: Apache Maven 3.6.3 gcc version: gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0

Environment Configuration:

Number of CPUs: 20
Amount of Memory: 9GB
Disk Space: 512GB

### CODE REVIEW

**Comment by lintool** [comment]
Please continue along the onboarding path.

**Comment by lintool** [comment]
Please collapse all your other PRs into this one, so we don't have a separate PR for each file.

**Comment by kerkathy** [comment]
Had the same issue on Ubuntu 22.04.2 LTS after following installation guide

**Comment by Stefan824** [comment]
Will do. Sorry for the inconvenience.

Best,
Stefan Min

Jimmy Lin ***@***.***> 于2023年10月15日周日 09:51写道：

**Comment by Stefan824** [comment]
> Hi, I'm not sure if here's the correct place to ask but I had the same error on Ubuntu 22.04.2 LTS after following installation guide. Any ideas on solving this?

Should it be the same error I encountered, it could be resolved by downgrading faiss to the 2021 version as stated above in the description of the error.

**Comment by kerkathy** [comment]
> Should it be the same error I encountered, it could be resolved by downgrading faiss to the 2021 version as stated above in the description of the error.

I see, that method worked indeed. I appreciate your reply! :)

**Comment by Stefan824** [comment]
> > Should it be the same error I encountered, it could be resolved by downgrading faiss to the 2021 version as stated above in the description of the error.
> 
> I see, that method worked indeed. I appreciate your reply! :)

Good to know my solution helped.

**Comment by lintool** [comment]
@Stefan824 - if you still want your PR merged, please fix conflicts and make sure reproduction entries are chronologically sorted.

File: docs/conceptual-framework.md [hunk 1]
```diff
@@ -287,5 +287,6 @@ Before you move on, however, add an entry in the "Reproduction Log" at the botto
...
 + Results reproduced by [@ksunisth](https://github.com/ksunisth) on 2023-09-27 (commit [`142c774`](https://github.com/castorini/pyserini/commit/142c774a303c906ee245913bc7e714b165074b77))
 + Results reproduced by [@maizerrr](https://github.com/maizerrr) on 2023-10-01 (commit [`bdb9504`](https://github.com/castorini/pyserini/commit/bdb9504b1757ab88247924b55a8fde3e5c1a3d20))
++ Results reproduced by [@Stefan824](https://github.com/stefan824) on 2023-10-04 (commit [`4f3da10`](https://github.com/castorini/pyserini/commit/4f3da10b99341d0bc2729590c23d9f1654d8ee37))
 + Results reproduced by [@shayanbali](https://github.com/shayanbali) on 2023-10-13 (commit [`f1d623c`](https://github.com/castorini/pyserini/commit/f1d623cdcb12c3083ff1db8aed4b84e81951a18c))
-+ Results reproduced by [@gituserbs](https://github.com/gituserbs) on 2023-10-18 (commit [`f1d623c`](https://github.com/castorini/pyserini/commit/f1d623cdcb12c3083ff1db8aed4b84e81951a18c))
++ Results reproduced by [@gituserbs](https://github.com/gituserbs) on 2023-10-18 (commit [`f1d623c`](https://github.com/castorini/pyserini/commit/f1d623cdcb12c3083ff1db8aed4b84e81951a18
```
Comment (lintool) [inline docs/conceptual-framework.md:292]: you introduced a typo here - close parens.

**Comment by lintool** [comment]
Please be more careful and detailed oriented.

File: docs/conceptual-framework.md (hunk 1 above)
Comment (Stefan824) [reply docs/conceptual-framework.md]: So sorry for the extra trouble! It is now fixed. Will be more careful and detail-oriented in the future.

File: docs/conceptual-framework.md [hunk 2]
```diff
@@ -287,5 +287,6 @@ Before you move on, however, add an entry in the "Reproduction Log" at the botto
...
 + Results reproduced by [@ksunisth](https://github.com/ksunisth) on 2023-09-27 (commit [`142c774`](https://github.com/castorini/pyserini/commit/142c774a303c906ee245913bc7e714b165074b77))
 + Results reproduced by [@maizerrr](https://github.com/maizerrr) on 2023-10-01 (commit [`bdb9504`](https://github.com/castorini/pyserini/commit/bdb9504b1757ab88247924b55a8fde3e5c1a3d20))
++ Results reproduced by [@Stefan824](https://github.com/stefan824) on 2023-10-04 (commit [`4f3da10`](https://github.com/castorini/pyserini/commit/4f3da10b99341d0bc2729590c23d9f1654d8ee37))
 + Results reproduced by [@shayanbali](https://github.com/shayanbali) on 2023-10-13 (commit [`f1d623c`](https://github.com/castorini/pyserini/commit/f1d623cdcb12c3083ff1db8aed4b84e81951a18c))
-+ Results reproduced by [@gituserbs](https://github.com/gituserbs) on 2023-10-18 (commit [`f1d623c`](https://github.com/castorini/pyserini/commit/f1d623cdcb12c3083ff1db8aed4b84e81951a18c))
++ Results reproduced by [@gituserbs](https://github.com/gituserbs) on 2023-10-18 (commit [`f1d623c`](https://github.com/castorini/pyserini/commit/f1d623cdcb12c3083ff1db8aed4b84e81951a18))
```
Comment (lintool) [inline docs/conceptual-framework.md:292]: you are still missing a `c` at the end.

File: docs/experiments-nfcorpus.md [hunk 3]
```diff
@@ -214,5 +214,6 @@ Before you move on, however, add an entry in the "Reproduction Log" at the botto
...
 + Results reproduced by [@ksunisth](https://github.com/ksunisth) on 2023-09-27 (commit [`142c774`](https://github.com/castorini/pyserini/commit/142c774a303c906ee245913bc7e714b165074b77))
 + Results reproduced by [@maizerrr](https://github.com/maizerrr) on 2023-10-01 (commit [`bdb9504`](https://github.com/castorini/pyserini/commit/bdb9504b1757ab88247924b55a8fde3e5c1a3d20))
++ Results reproduced by [@Stefan824](https://github.com/stefan824) on 2023-10-04 (commit [`4f3da10`](https://github.com/castorini/pyserini/commit/4f3da10b99341d0bc2729590c23d9f1654d8ee37))
 + Results reproduced by [@shayanbali](https://github.com/shayanbali) on 2023-10-13 (commit [`f1d623c`](https://github.com/castorini/pyserini/commit/f1d623cdcb12c3083ff1db8aed4b84e81951a18c))
-+ Results reproduced by [@gituserbs](https://github.com/gituserbs) on 2023-10-19  (commit [`f1d623c`](https://github.com/castorini/pyserini/commit/f1d623cdcb12c3083ff1db8aed4b84e81951a18c))
++ Results reproduced by [@gituserbs](https://github.com/gituserbs) on 2023-10-19  (commit [`f1d623c`](https://github.com/castorini/pyserini/commit/f1d623cdcb12c3083ff1db8aed4b84e81951a18c))
```
Comment (lintool) [inline docs/experiments-nfcorpus.md:219]: There's an extra space after the `2023-10-19` date here - I don't think you introduced the issue... but please fix.

**Review by lintool — APPROVED** [review]

**Pull request merged**